src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -115,13 +116,22 @@ renderError err =
 
 displayUserError :: UserError -> Box
 displayUserError e = case e of
-  PackageManifestNotFound ->
-    para (
-      "The package manifest file was not found. Please create one, or run " ++
-      "`pulp init`."
-      )
+  PackageManifestNotFound path -> do
+    vcat
+      [ para "The package manifest file was not found:"
+      , indented (para path)
+      , spacer
+      , para "Please create one."
```

Comment:
Should this error indicate that either a `bower.json` or `.purs.json` file is expected?